OneCommunityGlobal/HighestGoodNetworkApp
# Shereen unit test weekly summary modal

# Description
Unit test weekly summary modal


## How to test:
1. check into current branch
2. do `npm install` and `npm test WeeklySummaryModal.test.js` to run this PR locally
3. Make sure all the tests are passed.

## Screenshots or videos of changes:

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/7041610/945c7c11-472d-42fd-abc3-b99aed9528c6)
